test/client/2DPrinting.js

```diff
@@ -60,6 +67,7 @@ describe('2DPrinting', () => {
         });
     });
     it('Should return statuscode 400 when the RPC fails', done => {
```

Comment:
add a test case for if the logging api is down that the client will return `BAD_REQUEST`, see [this example](https://github.com/SCE-Development/SCE-RPC/blob/441e7b5209bfa74185fcfbdf3de650ade171ce02/test/client/LedSign.js#L160)